compiler_gym/service/proto/Proto.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <functional>
+#include <string>
+#include <unordered_map>
+
+#include "compiler_gym/service/proto/compiler_gym_service.pb.h"
+
+namespace compiler_gym {
+
+struct SpaceContainsEventChecker {
```

Comment:
This needs docstrings

Reply:
Done.

Reply:
I added also a `proto.rst` file to the documentation.

Reply:
Thanks!